packages/language-server/src/utils.ts

```diff
@@ -294,3 +294,14 @@ export function getAstroInstall(basePaths: string[]): AstroInstall | undefined {
 		},
 	};
 }
+
+export function getWorkspacePnpPath(workspacePath: string): string | null {
+	const possiblePath = resolve(workspacePath, '.pnp.cjs');
+
+	try {
+		require.resolve(possiblePath);
```

Comment:
First try was using `fs.existsSync`, but `esbuild` was complaining about `node` environment in browser build, so I decided to use that method.